lib/date_time/date_time.c

```diff
@@ -291,33 +300,34 @@ static void new_date_time_get(void)
 
 		LOG_DBG("Current time not valid");
 
-#if defined(CONFIG_DATE_TIME_NTP)
-		LOG_DBG("Fallback on NTP server");
+#if defined(CONFIG_DATE_TIME_MODEM)
+		LOG_DBG("Fallback on cellular network time");
```

Comment:
I'm tempted to get rid of "fallback" here as this is really the first thing we are trying and gives an impression that we already tried something else. What if we just say: `Next trying to obtain time from cellular network`?
And use same wording for NTP although that is a fallback when modem time is configured ON. When it's not on, even NTP is not really a fallback.
@simensrostad What do you think?

Reply:
That works!

Reply:
What about something like:

`Getting time from cellular network`
`Time from cellular network obtained`
`Did not get time from cellular network. This is normal blah blah...`

`Getting time from NTP server`
`Time from NTP server obtained`
`Did not get time from NTP server`

`Did not get time from any time source` OR
`Did not get time from any configured time source`

Reply:
Here is how it looks no if you don't get the time:
```
[00:00:00.259,582] <dbg> date_time.date_time_register_handler: Registering handler 0x1bedd
[00:00:00.823,974] <dbg> date_time.new_date_time_get: Updating date time UTC...
[00:00:00.823,974] <dbg> date_time.current_time_check: Date time never set
[00:00:00.824,005] <dbg> date_time.new_date_time_get: Current time not valid
[00:00:00.824,005] <dbg> date_time.new_date_time_get: Getting time from cellular network
[00:00:00.830,627] <dbg> date_time.time_modem_get: Could not get cellular network time, error: -8
[00:00:00.830,657] <dbg> date_time.new_date_time_get: Did not get time from cellular network. This is normal as some cellular networks don't provide it.
[00:00:00.830,657] <dbg> date_time.new_date_time_get: Getting time from NTP server
[00:00:00.832,031] <dbg> date_time.time_NTP_server_get: Not connected to LTE, skipping NTP UTC time update
[00:00:00.832,061] <dbg> date_time.new_date_time_get: Did not get time from NTP server
[00:00:00.832,092] <dbg> date_time.new_date_time_get: Did not get time from any time source
```

Reply:
And here is when you get the time:
```
[00:02:19.315,063] <dbg> date_time.new_date_time_get: Updating date time UTC...
[00:02:19.315,063] <dbg> date_time.current_time_check: Current date time too old
[00:02:19.315,093] <dbg> date_time.new_date_time_get: Current time not valid
[00:02:19.315,093] <dbg> date_time.new_date_time_get: Getting time from NTP server
[00:02:19.329,162] <dbg> date_time.time_NTP_server_get: Not connected to LTE, skipping NTP UTC time update
[00:02:19.329,193] <dbg> date_time.new_date_time_get: Did not get time from NTP server
[00:02:19.329,193] <dbg> date_time.new_date_time_get: Getting time from cellular network
[00:02:19.329,895] <dbg> date_time.time_modem_get: Response from modem: +CCLK: "21/09/27,12:19:48"

[00:02:19.329,925] <dbg> date_time.new_date_time_get: Time from cellular network obtained
```